_practicalities/intro.md

```diff
@@ -332,21 +341,21 @@ and that you will hear about. This is an attempt to acknowledge them, and
 explain why they can't work and what are their drawbacks before you attempt to
 implement them.
 
-### Use a meta-package
+### Don't use a meta-package
 
 It is possible to release a meta-package that has _virtually_ no code and relies
 on a conditional dependency to install its actual core code on the user system.
 For example, Frob-6.0 could be a meta-package which depends on
 Frob-real-py2 on Python < 3.0, and Frob-real-py3 on Python ≥ 3.4. While
 this approach is _doable_ this can make imports confusing.
 
-## Depend on setuptools
+## Don't depend on setuptools
```

Comment:
I think this one is 

```suggestion
## Do depend on setuptools>=24.3
```

Maybe ?

Reply:
How about `Don't depend on setuptools unless version >=24.3`

So that it's all a list of not to dos?

Reply:
sounds good updated suggestion.

Reply:
buildscript {
    repositories {
        // Gradle 4.1 and higher include support for Google's Maven repo using
        // the google() method. And you need to include this repo to download
        //